pytest.ini

```diff
@@ -23,8 +23,8 @@ addopts =
   -p pytest_cov
 
   # `pytest-cov`:
-  --cov
-  --cov-context=test
+  --cov=yarl/
+  --cov=tests/
```

Comment:
No, this change is incorrect. The config must be in `.coveragerc` to enable it to be reused outside pytest.